src/@utils/aquarius/index.ts

```diff
@@ -56,6 +68,39 @@ export function getFilterTerm(
   }
 }
 
+export function getFilter(...args: any[]) {
+  let filters = []
+  if (typeof args[0] === 'object') {
+    args[0].forEach((arg) => {
```

Comment:
suggestion(blocking): In general prefer usage of `for...of` over `forEach` for readability and performance. More context: https://biomejs.dev/linter/rules/no-for-each/ 

This comment applies to all places in the code where `forEach` is being used.